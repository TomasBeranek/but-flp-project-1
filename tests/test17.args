-i file1.in file2.in
