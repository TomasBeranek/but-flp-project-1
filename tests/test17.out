ERROR: Invalid number of arguments (max 2).
